cypress/integration/analytics.test.js

```diff
@@ -201,11 +211,13 @@ describe('search event', () => {
     cy.visit(pages.home, options)
     cy.waitForHydration()
 
-    cy.getById('store-input').click().type('shirt')
-    cy.getById('store-button')
-      .click()
-      .then(() => {
-        dataLayerHasEvent('search')
-      })
+    cy.get('form[data-store-search-input]').within(() => {
```

Comment:
cy.getById('store-search-input').within(...)

Reply:
Oh nice! Updated it [here](https://github.com/vtex-sites/base.store/pull/284/commits/507dd9762f408289f5dd070f985d89bc5153b0bf).